### PR TITLE
Onboardingv2

### PR DESCRIPTION
**Pull request by GuillaumeDSM**
requires https://github.com/Drakkar-Software/OctoBot/pull/2213
image
image
image
image

### CODE REVIEW

File: Services/Interfaces/web_interface/models/configuration.py [hunk 1]
```diff
@@ -877,6 +923,10 @@ def get_full_exchange_list(remove_config_exchanges=False):
     return [exchange for exchange in full_exchange_list if constants.UPDATED_CONFIG_SEPARATOR not in exchange]
 
 
+def get_default_exchange():
+    return ccxt.async_support.binance.__name__
```
Comment (Herklos) [inline Services/Interfaces/web_interface/models/configuration.py:927]: Shouldn't we use the default profile exchange?

Comment (GuillaumeDSM) [reply Services/Interfaces/web_interface/models/configuration.py:927]: We do already, this is if there is no enabled exchange in the current profile

Comment (Herklos) [reply Services/Interfaces/web_interface/models/configuration.py:927]: 👍

**Review by Herklos — APPROVED** [review]
👍

**Pull request merged**